### PR TITLE
Fix blatant typos

### PR DESCRIPTION
**Pull request by 0x2b3bfa0**
_[...] **vi** punisca il sacro nume che dell'India è domator._ 😉

### CODE REVIEW

File: docs/source/index.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ General index <genindex>
 Module index <modindex>
 Search <search>
 
-[Source code](https://github.com/DIDONEproject/musif/)
```
Comment (0x2b3bfa0) [inline docs/source/index.md:25]: Sphynx `toctree` also uses [reStructuredText](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links), not Markdown.

**Review by martianbug — APPROVED** [review]
Everything looks good! Thank you for your contribution :)

**Comment by 0x2b3bfa0** [comment]
@martianbug, note that, as an external contributor, I can't merge the pull request.

**Comment by martianbug** [comment]
Yes, sorry, I was about to merge. Just in case, for future contributions try to merge them on the develop branch :) Thanks! I'll merge now

**Comment by 0x2b3bfa0** [comment]
> _Just in case, for future contributions try to merge them on the develop branch_

Thanks! Sorry, I didn't notice you were using [git-flow](https://nvie.com/posts/a-successful-git-branching-model/). 🙈

**Pull request merged**